virtool/tasks/data.py

```diff
@@ -141,3 +145,43 @@ async def remove(self, task_id: int):
 
             session.delete(task)
             await session.commit()
+
+    async def add(self, task_class: Type[Task], context: dict = None):
```

Comment:
### Naming
Rename `add()` to `create()` to realign with rest of data layer.

### Use `Optional`
```suggestion
    async def add(self, task_class: Type[Task], context: Optional[Dict] = None):
```

https://docs.python.org/3/library/typing.html#typing.Optional